src/DynamicData/Cache/ObservableCacheEx.VirtualiseAndPage.cs

```diff
@@ -0,0 +1,191 @@
+﻿// Copyright (c) 2011-2023 Roland Pheasant. All rights reserved.
+// Roland Pheasant licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for full license information.
+
+using System.Reactive.Linq;
+using DynamicData.Cache.Internal;
+
+namespace DynamicData;
+
+/// <summary>
```

Comment:
I have included Virtualize, Page and Top operators here. Maybe overkill to have one file per operator.  Thoughts?

Reply:
Me, I would probably stick with one operator per-file, just for clarity.